salt/modules/vmctl.py

```diff
@@ -11,11 +11,14 @@
     target machine.
 """
 
+
 import logging
 import re
 
+# Imoprt salt libs:
 import salt.utils.path
 from salt.exceptions import CommandExecutionError, SaltInvocationError
+from salt.ext.six.moves import zip
 
```

Comment:
This port should go, since you're editing the file.

Reply:
removed! @s0undt3ch :sparkles: